db/patch-fix-kafka-producer-booleans.sql

```diff
@@ -0,0 +1,29 @@
+-- You should not modify if this have pushed to Github, unless it does serious wrong with the db.
+BEGIN TRANSACTION;
+
+-- Rename COLUMNs to another one (suffixed by `_old`)
+ALTER TABLE monitor
+    RENAME COLUMN kafka_producer_ssl TO kafka_producer_ssl_old;
+
+ALTER TABLE monitor
+    RENAME COLUMN kafka_producer_allow_auto_topic_creation TO kafka_producer_allow_auto_topic_creation_old;
+
+-- Add correct COLUMNs
+ALTER TABLE monitor
+    ADD COLUMN kafka_producer_ssl BOOLEAN default 0 NOT NULL;
+
+ALTER TABLE monitor
+    ADD COLUMN kafka_producer_allow_auto_topic_creation BOOLEAN default 0 NOT NULL;
+
+-- Set bring old values from `_old` COLUMNs to correct ones
+UPDATE monitor set kafka_producer_allow_auto_topic_creation = monitor.kafka_producer_allow_auto_topic_creation_old;
```

Comment:
I think the problem is here, it try to set `VARCHAR(255)` and `null` into a BOOLEAN column.

Reply:
I will check that out, but I have tested that, it should be Ok since only possible values are 0 and 1, also we have default value for the field and null value should work

Reply:
@louislam Hi, Found the problem I will fix that....

Reply:
@louislam fixed in https://github.com/louislam/uptime-kuma/pull/3984